677 Add hybrid programming example for bundle (#678)
Project-MONAI/tutorials: modules/bundle/README.md

# MONAI bundle
This folder contains the get_started tutorial and concrete training / inference examples for MONAI bundle features.

### [spleen segmentation](./spleen_segmentation)
A bundle example for volumetric (3D) segmentation of the spleen from CT image.
### [customize component](./custom_component)

> Example shows the use case that brings customized python component(like: transform, network, metrics, etc.) in the `train` config.

Reviewer: ```suggestion
Example shows the use cases of bringing customized python components, such as transform, network, and metrics, in a configuration-based workflow.
```

Author: updated in the latest commit.

Thanks.
